projects/kit/components/routable-dialog/generate-dialogable-route.ts

```diff
@@ -0,0 +1,26 @@
+import {Type} from '@angular/core';
+import {Route} from '@angular/router';
+import {TuiDialogOptions} from '@taiga-ui/core';
+
+import {TuiRoutableDialogComponent} from './routable-dialog.component';
+
+export function tuiGenerateDialogableRoute<I>(
+    component: Type<any>,
+    {path, ...dialogOptions}: Partial<TuiDialogOptions<I>> & {path?: string} = {},
```

Comment:
Can it be just `{path = ``, ...dialogOptions{`?

Reply:
Looks better. Refactored